tests/r/FormatRBearTest.py

```diff
@@ -129,3 +139,9 @@
     valid_files=(good_file_6,),
     invalid_files=(bad_file_3,),
     settings={'r_max_expression_length': '25'})
+
+FormatRBearDefaultSettingsTest = verify_local_bear(
+    FormatRBear,
+    valid_files=(good_file_7, good_file_4),
+    invalid_files=(bad_file_3,),
+    settings={'r_keep_blank_lines': 'true'})
```

Comment:
as it's a default test, you maybe want to omit `settings`^^ (especially `r_keep_blank_lines` doesn't do anything in fact :3)